src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValues.ts

```diff
@@ -15,7 +15,7 @@ export interface IDynamicFacetValueCreatorKlass {
 
 export class DynamicFacetValues implements IDynamicFacetValues {
   private facetValues: IDynamicFacetValue[];
-  private list = $$('ul', { className: 'coveo-dynamic-facet-values' }).el;
+  private list = $$('ul', { className: 'coveo-dynamic-facet-values', role: 'group' }).el;
```

Comment:
Unless we also change the role of all `li` elements, I don't think we should replace the `ul` element's default role. And even if we do, I don't see a much more appropriate role. `menu` and `menuitem` isn't so bad, in Mozilla's example, but I don't think I would consider facets a menu.

Reply:
Agreed, I wouldn't consider the facets a menu either ... I just got an idea. I will remove the `role` override and add an `aria-labelledby`. The screen reader will say: `<facet header> list`